.devcontainer/devcontainer.json

```diff
@@ -2,7 +2,9 @@
   "name": "TypeSpec Dev Container",
   "image": "mcr.microsoft.com/devcontainers/typescript-node:dev-20",
   // Default is 2 cores which is too slow for this codebase
-  "hostRequirements": {"cpus": 4},
+  "hostRequirements": {
```

Comment:
Revert formatting changes to all `*.json` files outside `specification`

Reply:
Done